CenterForOpenScience/ember-osf-web
# [ENG-517] Use schemablocks for overview page

- Ticket: [ENG-517]
- Feature flag: n/a

## Purpose

Use schema blocks for overview page.

## Summary of Changes

Added a new `RegistrationFormViewSchemaBlocks` component.


## QA Notes
Make sure the registration overview page works, including the navigation widget. This PR also changed the read-only file widget (the list of file names with link on both the review page and the overview page), so please also check that it works.

[ENG-517]: https://openscience.atlassian.net/browse/ENG-517
